### ISSUE TITLE
Bug - [table] - table column sorting is not accessible

### ISSUE THREAD

**Issue by Aishwarya-Urne** [patternfly/patternfly-react#11986]
This is an issue with
Patternfly 5
Patternfly 6

**Describe the problem**
The issue is with component table -> table-columns

**How do you reproduce the problem?**

1. Reach to the component Table -> Sortable & wrapping headers.
2. Start the voiceover and try to read the table column.
3. When we reach to the sorting option with Control+Option+Space it sorts the column data but does not read out whether the column data is sorted in ascending/descending order.

**Expected behavior**
It should read out whether the column data is sorted in ascending/descending order when we are using voiceover or any other screenreader tool. It is needed to make the table accessible.

**Screenshots**

<img width="3000" height="1644" alt="Image" src="https://github.com/user-attachments/assets/bb94ca62-0728-4eed-8388-8fe73e269886" />

**What is your environment?**
 - iOS

**Comment by thatblindgeye** [comment]
Since we have a couple other tbale issues that relate to this, I will close this out similar to the core issue that was opened https://github.com/patternfly/patternfly/issues/7786. @Aishwarya-Urne if you have any other thoughts to see this reopened please let us know
